tests/integration/test_keeper_watches/test.py

```diff
@@ -100,3 +100,22 @@ def assert_watch_triggers(expected_count):
     destroy_zk_client(node_zk)
     node_zk = None
     time.sleep(1)
+
+
+def test_ephemeral_watch_session_close(started_cluster):
+    wait_nodes()
+
+    zk = get_fake_zk(node.name)
+
+    watch_triggered = False
+
+    def watch_callback(event):
+        watch_triggered = True
```

Comment:
Didn't you want to assert that the watch is not triggered? If not, then maybe it can be removed from the test.
```suggestion

    def watch_callback(event):
        pass
```